UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/ModelStructureCheck.cs

```diff
@@ -0,0 +1,119 @@
+﻿using NUnit.Framework;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.FunctionalTests.Models;
+
+namespace UKHO.ExchangeSetService.API.FunctionalTests.Helper
+{
+    public static class ModelStructureCheck
+    {
+        private static TestConfiguration Config { get; set; }
+        static ModelStructureCheck()
+        {
+            Config = new TestConfiguration();
+        }
+        public static async Task CheckModelStructureForSuccessResponse(this HttpResponseMessage apiresponse)
+        {
+            var apiresponsedata = await apiresponse.ReadAsTypeAsync<ExchangeSetResponseModel>();
```

Comment:
Please can we maintain a consistent naming convention through our tests.

```suggestion
        public static async Task CheckModelStructureForSuccessResponse(this HttpResponseMessage apiResponse)
        {
            var apiResponseData = await apiResponse.ReadAsTypeAsync<ExchangeSetResponseModel>();
```

Reply:
Done changes Thanks.